packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -865,6 +880,10 @@ abstract class SortableTable<T> {
   void sortData(ColumnData column, SortDirection direction);
 }
 
+abstract class SelectableTable<T> {
+  T _selectedNode;
+}
+
```

Comment:
The idea was to consolidate selection logic so it could be reused by flat tables and tree tables. I had to move the actual implementation into FlatTableState, though, so it's not useful right now.